package.json

```diff
@@ -3,7 +3,7 @@
   "version": "2.3.0",
   "license": "Apache License 2.0",
   "config": {
-    "webservice_version": "1.9.0-beta.0"
+    "webservice_version": "1.9.1"
```

Comment:
Now that this is on 1.9.2 hotfix branch, shouldn't this be 1.9.2? Which isn't out yet? This is a broader question, but how are we suppose to handle this? As soon as we do the 1.9.2 web service release, update this to point to 1.9.2?

Also I had a comment about CircleCI artifacts, which I thought take precedence over this, but Gary approved, so I'm not sure.

Reply:
I thought the run-webservice-script.sh would use this webservice version as long as it was commented correctly? Unless you meant something else.

And yeah, I don't see a tag for 1.9.2. I guess bare minimum we make sure it was done before release to Dockstore? I don't know where in the release process Gary has those instructions.

Reply:
> I thought the run-webservice-script.sh would use this webservice version as long as it was commented correctly? Unless you meant something else.

You're right; I was looking at a branch that was pointing at CircleCI, but this branch is using the artifactory one.